src/main/scala/io/citrine/lolo/bags/BaggedResult.scala

```diff
@@ -60,21 +59,14 @@ case class BaggedSingleResult(
   private lazy val expected = treePredictions.sum / treePredictions.length
   private lazy val treeVariance: Double = treePredictions.map(x => Math.pow(x - expected, 2.0)).sum / treePredictions.length
 
-  /**
-    * Return jackknife-based variance estimates
-    *
-    * @return uncertainty of each prediction
-    */
-  override def getUncertainty(): Option[Seq[Any]] = Some(Seq(scalarUncertainty))
+  override def getPredictionInterval(): Option[Seq[Double]] = Some(Seq(treeVariance))
+
+  override def getRootVariance(): Option[Seq[Double]] = Some(Seq(scalarUncertainty))
 
   private lazy val scalarUncertainty: Double = {
-    if (useJackknife) {
```

Comment:
We always use Jackknife now?

Reply:
We'll always use it to get the variance component of the error, yes.  We may want to preserve a way to not carry the `Nib` matrix around if we're never going to use it, but the laziness of this declaration prevents us from paying the runtime cost and I'm not particularly concerned about memory right now.